dotCMS/src/integration-test/java/com/dotcms/scheduler/SchedulerAPITest.java

```diff
@@ -0,0 +1,105 @@
+package com.dotcms.scheduler;
+
+import java.util.Random;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import com.dotcms.business.CloseDBIfOpened;
+import com.dotcms.util.IntegrationTestInitService;
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.common.db.DotConnect;
+import com.dotmarketing.db.DbConnectionFactory;
+import com.dotmarketing.util.Config;
+import com.dotmarketing.util.Logger;
+import com.dotmarketing.util.UUIDGenerator;
+import com.github.kagkarlsson.scheduler.Scheduler;
+import com.github.kagkarlsson.scheduler.SchedulerName;
+import com.github.kagkarlsson.scheduler.task.helper.OneTimeTask;
+import com.github.kagkarlsson.scheduler.task.helper.Tasks;
+
+public class SchedulerAPITest {
+    
+    @CloseDBIfOpened
+    @BeforeClass
+    public static void prepare() throws Exception {
```

Comment:
Codacy found an issue: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6006494)